Update concept-metrics.md (#78631)
Azure/azure-content: articles/spring-cloud/concept-metrics.md

>|----|----|----|------------|
>| system.cpu.usage | system.cpu.usage | Percent | Recent CPU usage for the whole system (Obsolete and don't suggest using it). This value is a double in the [0.0,1.0] interval. A value of 0.0 means that all CPUs were idle during the recent period of time observed, while a value of 1.0 means that all CPUs were actively running 100% of the time during the recent period being observed.|
>| process.cpu.usage | App CPU Usage Percentage | Percent | Recent CPU usage for the Java Virtual Machine process (Obsolete and don't suggest using it). This value is a double in the [0.0,1.0] interval. A value of 0.0 means that none of the CPUs were running threads from the JVM process during the recent period of time observed, while a value of 1.0 means that all CPUs were actively running threads from the JVM 100% of the time during the recent period being observed. Threads from the JVM include the application threads as well as the JVM internal threads.|

> >| App CPU Usage | | Percent | Recent CPU usage of the JVM process against the CPU allocated to this app, double type value between [0.0,1.0]. A value of 0.0 means that none of the CPUs were running threads from the JVM process during the recent period of time observed, while a value of 1.0 means that all CPUs were actively running threads from the JVM 100% of the time during the recent period being observed. Threads from the JVM include the application threads as well as the JVM internal threads.|

Reviewer: Should there be a value in the second column for this row?

```suggestion
>| App CPU Usage | | Percent | Recent CPU usage of the JVM process against the CPU allocated to this app. This value is a double in the [0.0,1.0] interval. A value of 0.0 means that none of the CPUs were running threads from the JVM process during the recent period of time observed, while a value of 1.0 means that all CPUs were actively running threads from the JVM 100% of the time during the recent period being observed. Threads from the JVM include the application threads as well as the JVM internal threads.|
```

Author: I was asked to keep it blank by my boss.

> >| PodCpuUsage | App CPU Usage | Percent | Recent CPU usage of the JVM process against the CPU allocated to this app, double type value between [0.0,1.0]. A value of 0.0 means that none of the CPUs were running threads from the JVM process during the recent period of time observed, while a value of 1.0 means that all CPUs were actively running threads from the JVM 100% of the time during the recent period being observed. Threads from the JVM include the application threads as well as the JVM internal threads.|

Reviewer 2: Please change "PodCpuUsage" to be "App CPU Usage", the original doc should be incorrect.

> >| PodCpuUsage | App CPU Usage | Percent | Recent CPU usage of the JVM process against the CPU allocated to this app, double type value between [0.0,1.0]. A value of 0.0 means that none of the CPUs were running threads from the JVM process during the recent period of time observed, while a value of 1.0 means that all CPUs were actively running threads from the JVM 100% of the time during the recent period being observed. Threads from the JVM include the application threads as well as the JVM internal threads.|

Reviewer 2: "App CPU Usage" -> make this empty.

> >| App CPU Usage (Deprecated) | | Percent | Deprecated metric of App CPU Usage. Please use new App CPU Usage instead|

Reviewer: Should there be a value in the second column for this row?

```suggestion
>| App CPU Usage (Deprecated) | | Percent | Deprecated metric of App CPU Usage. Use the new App CPU Usage metric instead.|
```

Author: I was asked to keep it blank by my boss.

> >| AppCpuUsage | App CPU Usage (Deprecated) | Percent | Deprecated metric of App CPU Usage. Please use new App CPU Usage instead|

Reviewer 2: "AppCpuUsage" -> "App CPU Usage (Deprecated)"

> >| App Memory Usage | | Percent | Recent memory usage of the JVM process against the memory allocated to this app, double type value between [0.0,1.0]. A value of 0.0 means that none of the memory was allocated by threads from the JVM process during the recent period of time observed, while a value of 1.0 means that all memory was allocated by threads from the JVM 100% of the time during the recent period being observed. Threads from the JVM include the application threads as well as the JVM internal threads.|

Reviewer: Should there be a value in the second column for this row?

```suggestion
>| App Memory Usage | | Percent | Recent memory usage of the JVM process against the memory allocated to this app. This value is a double in the [0.0,1.0] interval. A value of 0.0 means that none of the memory was allocated by threads from the JVM process during the recent period of time observed, while a value of 1.0 means that all memory was allocated by threads from the JVM 100% of the time during the recent period being observed. Threads from the JVM include the application threads as well as the JVM internal threads.|
```

Author: I was asked to keep it blank by my boss.

>| process.cpu.usage | App CPU Usage Percentage | Percent | Recent CPU usage for the Java Virtual Machine process (Obsolete and don't suggest using it). This value is a double in the [0.0,1.0] interval. A value of 0.0 means that none of the CPUs were running threads from the JVM process during the recent period of time observed, while a value of 1.0 means that all CPUs were actively running threads from the JVM 100% of the time during the recent period being observed. Threads from the JVM include the application threads as well as the JVM internal threads.|
>| AppCpuUsage | App CPU Usage | Percent | Recent CPU usage of the JVM process against the CPU allocated to this app, double type value between [0.0,1.0]. A value of 0.0 means that none of the CPUs were running threads from the JVM process during the recent period of time observed, while a value of 1.0 means that all CPUs were actively running threads from the JVM 100% of the time during the recent period being observed. Threads from the JVM include the application threads as well as the JVM internal threads.|

> >| AppMemoryUsage | App Memory Usage | Percent | Recent memory usage of the JVM process against the memory allocated to this app, double type value between [0.0,1.0]. A value of 0.0 means that none of the memory were allocated by threads from the JVM process during the recent period of time observed, while a value of 1.0 means that all memory were allocated by threads from the JVM 100% of the time during the recent period being observed. Threads from the JVM include the application threads as well as the JVM internal threads.|

Reviewer: ```suggestion
>| AppMemoryUsage | App Memory Usage | Percent | Recent memory usage of the JVM process against the memory allocated to this app, double type value between [0.0,1.0]. A value of 0.0 means that none of the memory was allocated by threads from the JVM process during the recent period of time observed, while a value of 1.0 means that all memory was allocated by threads from the JVM 100% of the time during the recent period being observed. Threads from the JVM include the application threads as well as the JVM internal threads.|
```
